### ISSUE TITLE
Default Regular Speed goes from 1 to 1.15?

### ISSUE THREAD

**Issue by Adulau2003** [WofWca/jumpcutter#183]
Is this an inherent limitation of the addon? I can go in increments of 0.05, but it starts at 1, then goes to 1.15 with no inbetween. Same rules backwards: Goes from 1 to 0.85.

**Comment by WofWca** [comment]
No, this is not an inherent limitation. This simply has to do with the "snap" that is applied to the slider. You can either reduce the range (increase the "min" value and reduce the "max" value), or you can utilize the shortcuts "S" and "D":

![image](https://github.com/user-attachments/assets/d8dd095f-aef0-4ffc-a7d6-cccb9daf53b2)

You'll have to set the value of the shortcut to `0.05` as well. Let me know if it works for you.
